posthog/api/person.py

```diff
@@ -208,6 +208,16 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> response.
 
         return Response({"results": serialized_actors, "next": next_url, "previous": previous_url})
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "delete_events",
+                OpenApiTypes.BOOL,
+                description="If true, all events associated with this person will be deleted",
```

Comment:
```suggestion
                description="If true, a task to delete all events associated with this person will be created and queued. The task does not run immediately but is processed on the cron pattern set by CLEAR_CLICKHOUSE_REMOVED_DATA_SCHEDULE_CRON",
```

Reply:
If this is user facing I think "CLEAR_CLICKHOUSE_REMOVED_DATA_SCHEDULE_CRON" will lead to confusion. 

Instead how about "The task does not run immediately but instead is run roughly once a week"